com.unity.render-pipelines.core/Editor/Converter/RenderPipelineConverter.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.Rendering
+{
+    // Might need to change this name before making it public
+    public abstract class RenderPipelineConverter
+    {
+        /// <summary>
+        /// Name of the converter.
+        /// </summary>
+        public abstract string name { get; }
+
+        /// <summary>
+        /// The information when hovering over the converter.
+        /// </summary>
+        public abstract string info { get; }
+
+        /// <summary>
+        /// A check if the converter is enabled or not. Can be used to do a check if prerequisites are met to have it enabled or disabled.
+        /// </summary>
+        public virtual bool Enabled()
```

Comment:
I would rename it to `IsEnabled` the method will be self explanatory. and I would make it a property instead.